site/public/js/server.js

```diff
@@ -276,7 +280,8 @@ function newUploadCourseMaterialsForm() {
     var form = $("#upload-course-materials-form");
 
     $('[name="existing-file-list"]', form).html('');
-    $('[name="existing-file-list"]', form).append('<b>'+JSON.stringify(files)+'</b>');
+    var addingStuff = $("<b></b>").text(JSON.stringify(files));
```

Comment:
That's not a very descriptive variable name.  Also, it would be better to use `const` here.